ui/common/CodeExample.tsx

```diff
@@ -1,14 +1,23 @@
 import { CopyButton } from 'ui'
 
-export default function CodeExample({ code, language }) {
+export default function CodeExample({ code, language, copy }: {code: string; language: string; copy?: boolean | string}) {
```

Comment:
Do we require the prop - `copy` to have string value?
I think in most cases the copy button will be required to copy the content within the `CodeExample` box.

Reply:
Good call, I had added this just to check if this would be a good solution instead of adding the additional logic for `expectedInput.userVariable` and `.value` but with this being added this feature isn't really necessary